scripts/google-free.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
One last thing, it might be sensible to add `set -e` to have it fail if any of the commands fail.